Source/JavaScriptCore/jit/GPRInfo.h

```diff
@@ -1173,7 +1187,7 @@ class PreferredArgumentImpl {
             GPRInfo::argumentGPR2, GPRInfo::argumentGPR3,
             GPRInfo::regT2,        GPRInfo::regT3,
             GPRInfo::regT4,        GPRInfo::regT5,
-            GPRInfo::regT6,        GPRInfo::regT7);
```

Comment:
What's up with this change?

Reply:
They are not correctly working with the current use. Just fixed.